sdk/tests/conformance2/textures/misc/tex-srgb-mipmap.html

```diff
@@ -60,16 +60,14 @@
 </script>
 <script>
 "use strict";
-var canvas;
 var wtu = WebGLTestUtils;
+var canvas = document.getElementById("example");
+var gl = wtu.create3DContext(canvas);
```

Comment:
Please change this to:

var gl = wtu.create3DContext(canvas, undefined, 2);

so that this test works correctly when visited just via the URL, and doesn't require the query argument "?webglVersion=2" to be passed.